hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/operations/DurablePutResultOperation.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.durableexecutor.impl.operations;
+
+import com.hazelcast.nio.Bits;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.Notifier;
+import com.hazelcast.spi.WaitNotifyKey;
+
+import java.io.IOException;
+
+public class DurablePutResultOperation extends AbstractDurableOperation implements Notifier {
```

Comment:
Could you please verify my understanding:

We don't have a backup operation here but we have backup operation for `DurableDisposeResultOperation`. Once we complete the task, we only update our local ring buffer with the result. If the user is notified about the result, it will get the response from our ring buffer, and dispose the task from the backup ring buffer. If a backup node becomes partition owner, it will execute all the non-disposed tasks in its ring buffer.